packages/models/src/index.ts

```diff
@@ -33,6 +33,12 @@ export {
 } from './lib/implementation/limits';
 export { MaterialIcon, materialIconSchema } from './lib/implementation/schemas';
 export { exists } from './lib/implementation/utils';
+export {
+  TableHeading,
+  tableHeadingSchema,
```

Comment:
It seems strange to export table heading and not the body. To keep it simple, I would only export the full table.

```suggestion
```